test/Main.hs

```diff
@@ -0,0 +1,28 @@
+{-# LANGUAGE OverloadedStrings   #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+module Main where
+
+import GitHub.Data.Name
+import Hit.Issue (parseOwnerRepo)
```

Comment:
According to our style guide, you should group imports into multiple groups and put a blank line between groups. Groups are:

1. Dependencies from Hackage
2. Dependencies from the project itself

Reply:
Didn't know about this. Thanks for pointing it out!